reloader/reloader.py

```diff
@@ -112,8 +112,8 @@ def reload_missing(modules, verbose):
 
 def reload_plugin(pkg_name):
     pkg_path = os.path.join(os.path.realpath(sublime.packages_path()), pkg_name)
-    plugins = [pkg_name + "." + os.path.splitext(f)[0]
-               for f in os.listdir(pkg_path) if f.endswith(".py")]
+    plugins = [pkg_name + "." + os.path.splitext(file)[0]
+               for file in os.listdir(pkg_path) if file.endswith(".py")]
```

Comment:
`file` is a function in python, I prefer sth else.

Reply:
Only in Python 2 though (where it's actually a type).